docs/source/tutorial/client.md

```diff
@@ -129,7 +129,10 @@ Open `src/index.js` and add the following lines of code:
 _src/index.js_
 
 ```js lines=1,4,6
```

Comment:
Just flagging that the additional three lines below requires that these line offsets (used for highlighting) will need to be updated.

I _think_ to:

```suggestion
 ```js lines=3,7,9
```